src/presence_analyzer/tests.py

```diff
@@ -53,6 +54,97 @@ def test_api_users(self):
         self.assertEqual(len(data), 2)
         self.assertDictEqual(data[0], {u'user_id': 10, u'name': u'User 10'})
 
+    def test_mean_time_weekday(self):
+        """
+        Test correct format of output data.
```

Comment:
you can use this description to anyfunction that return something, should tell me more when i read this